test/parallel/test-require-symlink.js

```diff
@@ -22,8 +24,8 @@ if (common.isWindows) {
   // On Windows, creating symlinks requires admin privileges.
   // We'll only try to run symlink test if we have enough privileges.
   exec('whoami /priv', function(err, o) {
-    if (err || o.indexOf('SeCreateSymbolicLinkPrivilege') == -1) {
-      console.log('Skipped: insufficient privileges');
+    if (err || !o.includes('SeCreateSymbolicLinkPrivilege')) {
+      common.skip('Skipped: insufficient privileges');
```

Comment:
Sure I can change. Do you want me to revert it back to console.log or just remove this line?